lib/pacemaker/pcmk_sched_utils.c

```diff
@@ -765,34 +765,3 @@ pcmk__create_history_xml(xmlNode *parent, lrmd_event_data_t *op,
     free(key);
     return xml_op;
 }
-
-/*!
- * \internal
- * \brief Check whether a colocation constraint should apply
- *
- * \param[in] rsc            Resource of interest (for logging)
- * \param[in] colocation     Colocation constraint to check
- * \param[in] promoted_only  If true, constraint applies if right-hand is promoted
- */
-bool
-pcmk__colocation_applies(pe_resource_t *rsc, pcmk__colocation_t *colocation,
-                         bool promoted_only)
```

Comment:
Alas, pcmk__colocation_applies, we hardly knew ye.

Reply:
lol